Sources/Orbit/Support/Previews/PreviewFormatting.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+extension CGFloat {
+
+    var formatted: String {
+        -0.001...0.001 ~= self.remainder(dividingBy: 1)
```

Comment:
Is that really more readable than `(-0.001...0.001).contains(self.remainder(dividingBy: 1))`? 🤔 

Or even `abs(self.remainder(dividingBy: 1)) <= 0.001`?

Reply:
Updated using the `abs` version